tests/driver_winc1500_netdev/README.md

```diff
@@ -0,0 +1,32 @@
+# WINC1500 Wifi Module Driver
+
+## How to use the test program
+
+### Debugging support
+
+To get more debugging messages while developing you can enable debugging mode
+by uncommenting `#CFLAGS += -DLOG_LEVEL=LOG_DEBUG` in `Makefile`.
+
+## WINC1500 Xplained Pro Pinmap
+
+### ATWINC1500 Connection
+
+| Pin | Function  |
+|:---:|-----------|
+| 5   | RESET_N   |
+| 6   | WAKE      |
+| 9   | IRQ_N     |
+| 10  | CHIP_EN   |
+| 15  | SPI_SSN   |
+| 16  | SPI_MOSI  |
+| 17  | SPI_MISO  |
+| 18  | SPI_SCK   |
+| 19  | GND       |
+| 20  | VCC       |
+
+### ATECC508A Crypto device Connection
```

Comment:
Is this also available in the ATWINC1500 ? or only related to the atmel extension ?

Reply:
The ATECCx08 is something like a companion chip with ATWINC1500, although it is a separated chip. MKR1000 also has that chip (ATECC108) inside of the ATSAM25 module. I guess this crypto module is used for Atmel's built-in SSL socket implementation. So I thought it is worth noting here.

Reply:
ok